migration_steps/prepare_source_data/counts_verification/sql/count_casrec_source.sql

```diff
@@ -0,0 +1,384 @@
+(
+    -- client_nodebtchase
+    SELECT COUNT(*)
+    FROM casrec_csv.pat
+    WHERE casrec_csv.pat."Debt chase" != ''
+)+(
+    -- client_saarcheck
+    SELECT COUNT(*)
+    FROM casrec_csv.pat
+    WHERE casrec_csv.pat."SAAR Check" != ''
+)+(
+    -- client_special
+    SELECT COUNT(*)
+    FROM casrec_csv.pat
+    WHERE casrec_csv.pat."SIM" != ''
+        AND casrec_csv.pat."SIM" != '0'
+)+(
+    -- client_violent
+    SELECT COUNT(*)
+    FROM casrec_csv.pat
+    WHERE casrec_csv.pat."VWM" != ''
+        AND countverification.warning_violent_lookup(casrec_csv.pat."VWM") = 'Marker'
+)+(
+    -- p1_client_remarks
+    SELECT COUNT(*)
+    FROM casrec_csv.remarks rem
+    WHERE rem."Pri" = '1'
+)+(
+    -- deputy_special
+    SELECT COUNT(*)
+    FROM countverification.filtered_deps fd
+    INNER JOIN casrec_csv.deputy dep ON dep."Deputy No" = fd."Deputy No"
+    WHERE dep."SIM" != '' AND dep."SIM" != '0'
+)+(
+    -- deputy_violent
+    SELECT COUNT(*)
+    FROM countverification.filtered_deps fd
+    INNER JOIN casrec_csv.deputy dep ON dep."Deputy No" = fd."Deputy No"
+    WHERE countverification.warning_violent_lookup(dep."VWM") = 'Marker'
+)
+WHERE supervision_table = 'warnings';
+
+-- annual_report_logs
+UPDATE countverification.counts SET casrec_source =
+(
+    SELECT COUNT(*) FROM casrec_csv.account
+)
+WHERE supervision_table = 'annual_report_logs';
+
+-- visits
+UPDATE countverification.counts SET casrec_source =
+(
+    SELECT COUNT(*) FROM (
+        SELECT DISTINCT vis."casrec_row_id"
+        FROM casrec_csv.REPVIS vis
+--         INNER JOIN countverification.filtered_orders o ON o."Case" = vis."Case"
+    ) t1
+)
+WHERE supervision_table = 'visits';
+
+-- bonds
+UPDATE countverification.counts SET casrec_source =
+(
+    -- active
+    SELECT COUNT(*) FROM casrec_csv.order
+    WHERE casrec_csv.order."Ord Stat" != 'Open'
+    AND casrec_csv.order."Bond Rqd" = 'Y'
+)+(
+    -- dispensed
+    SELECT COUNT(*) FROM casrec_csv.order
+    WHERE casrec_csv.order."Ord Stat" != 'Open'
+    AND casrec_csv.order."Bond Rqd" = 'S'
+)
+WHERE supervision_table = 'bonds';
+
+-- feepayer_id
+UPDATE countverification.counts SET casrec_source =
+(
+    SELECT COUNT(*)
+    FROM countverification.filtered_orders o
+    LEFT JOIN casrec_csv.deputyship ds ON ds."Order No" = o."Order No"
+    WHERE ds."Fee Payer" = 'Y'
+)
+WHERE supervision_table = 'feepayer_id';
+
+-- timeline_event
+UPDATE countverification.counts SET casrec_source =
+(
+    SELECT COUNT(*) FROM casrec_csv.pat
+)
+WHERE supervision_table = 'timeline_event';
+
+-- person_timeline
+UPDATE countverification.counts SET casrec_source =
+(
+    SELECT COUNT(*) FROM casrec_csv.pat
+)
+WHERE supervision_table = 'person_timeline';
+
+-- supervision_level_log
+UPDATE countverification.counts SET casrec_source =
+(
+    SELECT COUNT(*)
+    FROM countverification.filtered_orders
+)
+WHERE supervision_table = 'supervision_level_log';
+
+-- finance_invoice_ad
+SET datestyle = "ISO, DMY";
```

Comment:
@devkokov could you please cast your eye over these as you can tell what should be in the WHERE clauses? You'll be a lot faster and more accurate than me.

Reply:
Where clauses look good